docker/localnet/script/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM ubuntu:22.04
```

Comment:
can you use a lightweight base image?

Reply:
Coinbase requires using Ubuntu 20 or higher as the base image, rather than Alpine or Debian.